### ISSUE TITLE
Uploaded/processed tracks not visible in user profile or on the map

### ISSUE THREAD

**Issue by ToeBee** [openstreetcam/openstreetview.org#128]
Possibly related to #127 since something was clearly messed up with my account. But After that issue got closed, a bunch of failed upload attempts suddenly appeared in my profile. I deleted them today. However there are still 7 or 9 traces that were successfully uploaded and processed but still have not shown up in my user profile or on the map.

One of them is [474549](http://openstreetcam.com/details/474549/0). Notice also that this track has a ridiculous number of points associated with it. Something clearly went wrong in processing. I'm hoping you can track down all the missing ones on your end but I probably still have the sequence file on my hard drive if you need more IDs.

**Comment by ToeBee** [comment]
I just uploaded two new sequences. They both showed in my profile while they were uploading and processing. However now that they are finished processing, they no longer show in my profile. They are [478899](http://openstreetcam.com/details/478899/0) and [478905](http://openstreetcam.com/details/478905/0)

And it looks like the point scoring is similarly messed up on these two as well.

**Comment by bogdan-telenav** [comment]
This are send to the backend guys to solve the problem. They now about this problem. Also we appreciate that you are an honest guy and don't want extra points :))

**Comment by ToeBee** [comment]
I've been gone for 2 weeks, taking 10s of thousands more pictures to upload. Is this fixed yet? Looking at my profile, I think it might be.

**Comment by ToeBee** [comment]
Well now the problem with points is opposite. Everything seems to be categorized as "10+" coverage even when there is only one or two other tracks. Example: Track [496223](http://openstreetcam.com/details/496223/0) which runs along a stretch of Interstate 70 between Oakley and Colby (See [here](http://openstreetcam.com/map/@39.22701737303788,-100.89834094047546,16z)) which only has one previous track (#[28948](http://openstreetcam.com/details/28948/0)) along it.

**Comment by ToeBee** [comment]
Looks like that last example is now correct.

**Comment by alexandrui-telenav** [comment]
All points related problems should be corrected. We found a bug that sometimes instead of updating a row in the DB the value was added to what was already there, hence the huge amount of points. Sorry about that, it should now be corrected for all tracks!

By the way, @ToeBee, what setup did you use for http://openstreetcam.com/details/496223/0 ? The heading seems to be very accurate which is unusual for a smartphone. 👍

**Comment by ToeBee** [comment]
It is a GoPro Hero 5 mounted to the hood of my car with the suction cup mount. Unfortunately, while the Hero 5 does have geotagging built in, it does not set the heading. But since I have it facing in the same direction for the entire sequence (straight forwards in this case) it is easy to set the heading with a script based on direction of travel before I upload.

![2017-05-13 18 56 46](https://user-images.githubusercontent.com/348948/28742103-9a4904f6-73ec-11e7-855f-8eabb56f7da0.jpg)

**Comment by alexandrui-telenav** [comment]
Got it, thanks for the details, seems like a great setup! So by direction of travel you mean setting the heading toward the next GPS position in the track, right? We're also playing with a few approaches server side, one would be the next location, other the direction of the street segment. 
Also, how much time would the GoPro record before running out of battery? 🤔

**Comment by ToeBee** [comment]
Yes. I actually use the `interpolate_direction.py` script from the Mapillary tools repo to do it. If I was shooting in a direction other than straight forward it would let me set an offset from the direction of travel as well.

The GoPro battery lasts about 2.5 hours in picture mode. I have a total of 4 batteries and a dual battery charger so I can keep going all day long :) I first started mounting the camera on the hood instead of inside the windshield last winter. It works pretty well however on this trip I did learn that the summer heat brings an additional complication: bugs. The end of this sequence is probably the worst example: http://openstreetcam.com/details/496225/8133 

When I'm on the bicycle, I have a very similar setup. I have a handle bar mount that holds both my Garmin bike computer and the GoPro hanging under it. 4 batteries are enough to get me through 80 mile touring days but I can also carry a USB battery pack to charge the camera batteries while riding. Example bicycle track: http://openstreetcam.com/details/470463/0
